src/lucky/renderable.cr

```diff
@@ -34,18 +35,28 @@ module Lucky::Renderable
   end
 
   private def handle_response(response : Lucky::Response)
+    log_response(response)
     response.print
   end
 
   private def handle_response(_response : T) forall T
-    {% raise <<-ERROR
+    {%
+      raise <<-ERROR
 
-     #{@type} returned #{T}, but it must return a Lucky::Response.
+      #{@type} returned #{T}, but it must return a Lucky::Response.
 
-     Try this...
-       ▸ Make sure to use a method like `render`, `redirect`, or `json` at the end of your action.
-       ▸ If you are using a conditional, make sure all branches return a Lucky::Response.
-    ERROR %}
+      Try this...
+        ▸ Make sure to use a method like `render`, `redirect`, or `json` at the end of your action.
+        ▸ If you are using a conditional, make sure all branches return a Lucky::Response.
+      ERROR
+    %}
+  end
+
+  private def log_response(response)
```

Comment:
I think adding a type restriction here might make it more clear what the response is: `response : Lucky::Response`. So people know it's not an `HTTP::Server::Response`

Reply:
Good call.